public/index.js

```diff
@@ -1,17 +1,23 @@
 const resultElement = document.querySelector(".gpt-response");
 const buttons = document.querySelectorAll("button");
 const lactoseIntolerant = document.querySelector(".lactose-intolerant");
+const vegan = document.querySelector(".vegan");
+console.log(vegan);
```

Comment:
`console.log()`s can be removed if you've established that the element has been selected using `querySelector()`. But I know that I'm often guilty of leaving them lying around when I create PRs lol